recipe/conda_build_config.yaml

```
@@ -507,6 +507,8 @@ libopentelemetry_cpp:
   - '1.13'
 libosqp:
   - 0.6.3
+libopenvino-dev:
```

```suggestion
libopenvino_dev:
```

this is a quirk of this pinning since(i think)  "-" is special in yaml